docs/spec/circuit-breaker/01_concepts.md

```diff
@@ -0,0 +1,16 @@
+# Concepts
+
+The intention of the circuit breaker is to have a contingency plan for a
+running network which maintains network liveness. This can be achieved through
+selectively "pausing" functionality of specific modules on a running network.
```

Comment:
should we also mention selectively disabling certain messages?

Reply:
and maybe disabling other functionality - begin / end blockers (debatable)

Reply:
Both of these points are mentioned further on